multi/imxrt-multi/uart.c

```diff
@@ -680,6 +680,8 @@ int uart_init(void)
 
 	uart_initPins();
 
+	const speed_t default_baud[] = { UART_BAUDRATES };
```

Comment:
```suggestion
	const speed_t default_baud[sizeof(uartConfig) / sizeof(uartConfig[0])] = { UART_BAUDRATES };
```
If you don't use _Static_asset(), an explicit size array declaration might be better in this case .
The same for `info[]` array.

Reply:
Added an issue (https://github.com/phoenix-rtos/phoenix-rtos-devices/issues/89) as this is not the scope of this PR